yellowbrick/features/pcoords.py

```diff
@@ -169,6 +200,37 @@ def draw(self, X, y, **kwargs):
         Called from the fit method, this method creates the parallel
         coordinates canvas and draws each instance and vertical lines on it.
         """
+        # Convert from dataframe
+        if is_dataframe(X):
+            X = X.as_matrix()
+
+        # Choose a subset of samples
+        if isinstance(self.sample, int):
+            if self.sample < 1:
+                raise YellowbrickValueError(
+                    "`sample` parameter of type `int` must be greater than 1"
+                )
+            n_samples = min([self.sample, len(X)])
+        elif isinstance(self.sample, float):
+            if self.sample <= 0 or self.sample > 1:
+                raise YellowbrickValueError(
+                    "`sample` parameter of type `float` must be between 0 and 1"
+                )
+            n_samples = int(len(X) * self.sample)
+        else:
+            raise YellowbrickTypeError(
+                "`sample` parameter must be int or float"
+            )
+        X = X[:n_samples, :]
+
+        # Normalize
+        if self.normalize is not None:
+            if self.normalize not in self.normalizers:
+                raise YellowbrickValueError(
```

Comment:
So, correct me if I'm way off here, but my understanding of the purpose of this and other transformers in the package is that these should produce visualizations via various methods, but they should *not* actually perform transformations on the data passed through the `transform` method.  So here, when we perform normalization, that normalization applies only to the way that we visualize the data, but we are not actually returning normalized data in the `transform` method. 

It seems to me that this is the desired behavior because there are probably plenty of situations when I would want to *visualize* my data in a normalized way but I would want to build the visualization into a pipeline where the data either 1) was not normalized or 2) was normalized/preprocessed by a different method.  In other words, the normalization that makes for the best diagnostic visualization may not be the same normalization that I actually want to use for my ML model.  

From looking at the existing transformers, it seems like this is the approach for everything except the `pca` visualizer, which does return PCA-transformed features. So, up to you which way you want the API to be headed.  I kind of like keeping the visualizers as visualizers and allowing the user to do the actual transformations by other means.